zipline/utils/run_algo.py

```diff
@@ -356,24 +332,8 @@ def run_algorithm(start,
     """
     load_extensions(default_extension, extensions, strict_extensions, environ)
 
-    non_none_data = valfilter(bool, {
-        'data': data is not None,
-        'bundle': bundle is not None,
-    })
-    if not non_none_data:
-        # if neither data nor bundle are passed use 'quantopian-quandl'
-        bundle = 'quantopian-quandl'
-
-    elif len(non_none_data) != 1:
-        raise ValueError(
-            'must specify one of `data`, `data_portal`, or `bundle`,'
-            ' got: %r' % non_none_data,
-        )
-
-    elif 'bundle' not in non_none_data and bundle_timestamp is not None:
-        raise ValueError(
-            'cannot specify `bundle_timestamp` without passing `bundle`',
-        )
+    if benchmark_returns is None:
+        benchmark_returns, _ = load_market_data(environ=environ)
```

Comment:
We may now be able to remove the treasury curve download from `load_market_data`, since we have been assuming a risk free rate of 0. (However, with treasury rates currently on the rise, we may want to rechallenge that assumption.)